packages/clay-date-picker/src/Helpers.ts

```diff
@@ -103,17 +106,36 @@ export function range({end, start}: {end: number; start: number}) {
 	);
 }
 
-/**
- * Helper function for getting certain props from `moment`.
- * This allows users to not have to import and use `moment` themselves.
- */
-export function getLocaleProps(locale: string) {
-	const localeData = moment.localeData(locale);
-
-	return {
-		dateFormat: localeData.longDateFormat('L'),
-		firstDayOfWeek: localeData.firstDayOfWeek(),
-		months: localeData.months(),
-		weekdaysShort: localeData.weekdaysShort(),
-	};
-}
+export const addMonths = (date: number | Date, months: number) => {
+	date = clone(date);
+
+	date.setMonth(date.getMonth() + months);
+
+	return date;
+};
```

Comment:
I don't think we should always export functions in the same way. You have both:

```js
export function a() {
  // ...
};
```

and:

```js
export const b() => {
  // ...
};
```

(FWIW, I prefer the former because of hosting.)

Reply:
Edit (sigh): of course I meant "I do think"...

Reply:
I created https://github.com/liferay/eslint-config-liferay/issues/184 to track enforcing this, with the recommendation that we go with `export function` — but feel free to comment there if you want to advocate the opposite (either way, we should enforce it).